README.md

```diff
@@ -1,6 +1,82 @@
 # stereo_based_tracking
 Stereo-based 3D Object Tracking for Autonomous Driving
 
+# Build
+
+## Setup
+If your base github directory is ~/w, then proceed as follows.
+
+```
+ cd ~/w
+```
+
+Clone the repos.
+```
+git clone https://github.com/shoaiblari/stereo_based_tracking.git
+```
+
+Clone the nuscenes-devkit
+```
+git clone https://github.com/nutonomy/nuscenes-devkit.git
+```
+
+Install the nuscenes dataset in directory such as `/cs231a/data/nuscenes`.
+
+Make sure have anaconda3 installed.
+
+Your `PATH` should have anaconda3/bin on it, and yoru PYTHONPATH should include
+the nuscenes-devkit. In the example below, the following two lines were added to
+the end of the .bashrc to make this happen.
+
+```
+export PATH=/home/ubuntu/anaconda3/bin:$PATH
+export PYTHONPATH="/home/slari1/w/nuscenes-devkit/python-sdk:${PYTHONPATH}"
+```
+logout and log back in.
+
+Run
+```
+conda init bash
+```
+
+Edit your .bashrc to add the following line at the end.
+```
+conda deactivate
+```
+
+Logout and log back in.
+
+Create the `pt` Conda environment.
+
+```
+conda create --name pt python=3.6
+conda activate pt
+```
+
+Install the nuscenes-devkit
+```
+cd ~/w//nuscenes-devkit
+pip install -r setup/requirements.txt
```

Comment:
pip3?

Reply:
If the Conda env is setup right, it pip will resolve to pip3.